header-validator/validate-json.js

```diff
@@ -3,7 +3,7 @@ const int64Regex = /^-?[0-9]+$/
 const hex128Regex = /^0[xX][0-9A-Fa-f]{1,32}$/
 
 const limits = {
-  maxAggregationKeys: 50,
+  maxAggregationKeys: 20,
```

Comment:
Could you clarify why we're making this change in this PR?

Reply:
Actually this is merged from another branch by mistake, will revert.